generate/template/astronomy.kt

```diff
@@ -2799,7 +2797,7 @@ private fun vsopDerivCalc(formula: VsopFormula, t: Double): Double {
     var tpower = 1.0        // t^s
     var dpower = 0.0        // t^(s-1)
     var deriv = 0.0
-    for ((s, series) in formula.series.withIndex()) {
+    formula.series.forEachIndexed { s, series ->
```

Comment:
This one actually can be faster, in my codes I only use .forEach / .forEachIndexed instead all `for` wherever possible as they are `inline` and don't I should be considered but that probably in this project, instead I liked to use `.forEachIndexed` in this case `.forEachIndexed` provides something that regular `for` can't easily.